reference/SimpleCalculator/SimpleCalculator.Shared/MainPageMarkup.cs

```diff
@@ -80,15 +81,15 @@ private TextBlock Equation(MainModel.MainViewModel vm)
             => new TextBlock()
             .Text(() => vm.Calculator.Equation)
             .AutoLayout(counterAlignment: AutoLayoutAlignment.End)
-            .Foreground(ThemeResource.Get<Brush>("OnSecondaryContainerBrush"))
-            .Style(StaticResource.Get<Style>("DisplaySmall"));
+            .Foreground(Theme.Brushes.OnSecondary.Container.Default)
+            .Style(Theme.Styles.TextBlock.DisplaySmall);
 
         private TextBlock Result(MainModel.MainViewModel vm)
             => new TextBlock()
             .Text(() => vm.Calculator.Output)
             .AutoLayout(counterAlignment: AutoLayoutAlignment.End)
-            .Foreground(ThemeResource.Get<Brush>("OnBackgroundBrush"))
-            .Style(StaticResource.Get<Style>("DisplayLarge"));
+            .Foreground(Theme.Brushes.OnBackground.Default)
+            .Style(Theme.Styles.TextBlock.DisplayLarge);
 
```

Comment:
I think there are other brushes used in the KeyPad and KeyPadButton methods that should be able to use the constants

Reply:
@pictos or @eriklimakc can you pick this up and update the remaining methods with the theme constants

Reply:
We were instructed to leave it as it is for now @nickrandolph. But we may go back to it later.